source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -593,6 +595,224 @@ For example: If there are 10 available pieces of Medium size, 15 of Large and 5
 <quantity>30</quantity>
 ~~~
 
+
+### Size Variation Attributes
+
+Size variations is an enhancement to the current XML feed, that introduces new nodes (children)
```

Comment:
Since the noun is plural, I think we should use 'Size variations are'. 
```suggestion
Size variations are an enhancement to the current XML feed, that introduce new nodes (children)
```
If we mean the concept of "size variations" maybe we should use quotation marks
```suggestion
"Size variations" is an enhancement to the current XML feed, that introduces new nodes (children)
```